astro/src/content/quickstarts/quickstart-php-web.mdx

```diff
@@ -0,0 +1,310 @@
+  lang="php"/>
+
+In addition to verifying login, the code at the top of the change page checks if the CSRF token is valid on POST requests. The CSRF token is hidden in the form in the HTML, `<input type="hidden" name="csrftoken" value="<?= $_SESSION["csrftoken"] ?>" />`. Although any attacker can make a POST request on behalf of a logged in user, a server will reject a GET request if it's not made from the same origin (URL). Including this CSRF token in your page requires an attacker to make a GET request to obtain it before making the POST request, which isn't possible.
+
+The `calculateChange()` method takes the amount given in the form and returns a `$state` array with the amount in nickels and pennies.
+
+The HTML at the bottom of the file displays a blank form when the page first loads (GET) or the result of the calculation when returning (POST).
+
+## Run The Application
+
+The application is already being served by Apache in Docker.
+
+Browse to the app at http://localhost:9012. Log in using `richard@example.com` and `password`. The change page allows you to enter a number.
+
+## Next Steps
+
+This quickstart is a great way to get a proof of concept up and running quickly, but to run your application in production, there are some things you're going to want to do.
+
+### FusionAuth Customization
+
+FusionAuth gives you the ability to customize just about everything to do with the user's experience and the integration of your application. This includes:
+
+* [Hosted pages](/docs/v1/tech/themes/) such as login, registration, email verification, and many more.
+* [Email templates](/docs/v1/tech/email-templates/email-templates).
+* [User data and custom claims in access token JWTs](/articles/tokens/jwt-components-explained).
+
+### Security
+
+* You may want to customize the [token expiration times and policies](/docs/v1/tech/oauth/#configure-application-oauth-settings) in FusionAuth.
+* Choose [password rules](/docs/v1/tech/core-concepts/tenants#password) and a [hashing algorithm](/docs/v1/tech/reference/password-hashes) that meet your security needs.
+
+### Tenant And Application Management
+
+* Model your application topology using [Applications](/docs/v1/tech/core-concepts/applications), [Roles](/docs/v1/tech/core-concepts/roles), [Groups](/docs/v1/tech/core-concepts/groups), [Entities](/docs/v1/tech/core-concepts/groups), and more.
+* Set up [MFA](/docs/v1/tech/guides/multi-factor-authentication), [Social login](/docs/v1/tech/identity-providers/), or [SAML](/docs/v1/tech/identity-providers/samlv2/) integrations.
+* Integrate with external systems using [Webhooks](/docs/v1/tech/events-webhooks/), [SCIM](/docs/v1/tech/core-concepts/scim), and [Lambdas](/docs/v1/tech/lambdas/).
+
+### {frontmatter.technology} Authentication
+
+- [The League OAuth overview](https://oauth2-client.thephpleague.com/usage/)
+- [Jerry Hopper's FusionAuth provider](https://github.com/jerryhopper/oauth2-fusionauth)
+
+## Troubleshooting
+
```

Comment:
Ran into same issue as I mentioned here: https://github.com/FusionAuth/fusionauth-site/pull/2577#pullrequestreview-1721442839

> In both the complete application and working through the quickstart, I sometimes get an error: Your browser sent a request that this server could not understand. Size of a request header field exceeds server limit.Opening it up in an incognito window solved it, so can you add that to the troubleshooting section?


Can you please add the same troubleshooting step.

Reply:
I have added the troubleshooting step